tests/conftest.py

```diff
@@ -36,7 +47,42 @@ def pytest_internalerror(excinfo):
         raise excinfo.value
 
 
+def get_model_system(
+    type: str = 'original',
+    positions: List[List[float]] = [[0, 0, 0], [0.5, 0.5, 0.5]],
+    chemical_symbols: List[str] = ['Ga', 'As'],
+    orbital_symbols: List[List[str]] = [['s'], ['px', 'py']],
+) -> ModelSystem:
+    """
+    Create a `ModelSystem` object with the given parameters.
+    """
+    model_system = ModelSystem()
+    atomic_cell = AtomicCell(type=type, positions=positions * ureg.meter)
```

Comment:
If you want to avoid conversions and stay with the default, you can leave out `ureg.meter`. It even comes across as less silly, i.e. a meter spacing between atoms :p

Reply:
It just helps understanding the units of these quantities, so I prefer to leave it (same for the others).